docs/Using-Fleet/MDM-macOS-setup.md

```diff
@@ -150,7 +150,79 @@ You should see the URL for your bootstrap package as the value for `mdm.macos_se
 
 ## macOS Setup Assistant
 
-> This feature is currently in development.
+When an end user unboxes their new Mac, or starts up a freshly wiped Mac, they're presented with the macOS Setup Assistant. Here they see panes that allow them to configure accessibility, appearance, and more.
+
+In Fleet, you can customize the macOS Setup Assistant by using an automatic enrollment profile.
+
+To customize the macOS Setup Assistant, we will do the following steps:
+
+1. Create an automatic enrollment profile
+2. Upload the profile to Fleet
+3. Test the custom macOS Setup Assistant
+
+### Step 1: create an automatic enrollment profile
+
+1. Download Fleet's example automatic enrollment profile by navigating to the example [here on GitHub](https://github.com/fleetdm/fleet/blob/main/mdm_profiles/setup_assistant.json) and clicking the download icon.
+
+2. Open the automatic enrollment profile and replace the `profile_name` key with your organization's name.
+
+3. View the the list of macOS Setup Assistant properties (panes) [here in Apple's Device Management documentation](https://developer.apple.com/documentation/devicemanagement/skipkeys) and choose which panes to hide from your end users.
+
+4. In your automatic enrollment profile, edit the `skip_setup_items` array so that it includes the panes you want to hide.
+
+### Step 2: upload the profile to Fleet
+
+Choose which team you want to add the automatic enrollment profile to.
+
+In this example, we'll add the automatic enrollment profile to the "Workstations (canary)" team. Only hosts that automatically enroll to this team will see the custom macOS Setup Assistant.
+
+2. Create a `workstations-canary-config.yaml` file:
+
+```yaml
+apiVersion: v1
+kind: team
+spec:
+  team:
+    name: Workstations (canary)
+    mdm:
+      macos_setup:
+        macos_setup_assistant: ./path/to/automatic_enrollment_profile.json
+    ...
+```
+
+Learn more about team configurations options [here](./configuration-files/README.md#teams).
+
+To customize the macOS Setup Assistant for hosts that automatically enroll to "No team," we'll need to create an `fleet-config.yaml` file:
+
+```yaml
+apiVersion: v1
+kind: config
+spec:
+  mdm:
+    macos_setup:
+      macos_setup_assistant: ./path/to/automatic_enrollment_profile.json
+  ...
+```
+
+Learn more about configuration options for hosts that aren't assigned to a team [here](./configuration-files/README.md#organization-settings).
+
+3. Add an `mdm.macos_setup.macos_setup_assistant` key to your YAML document. This key accepts a path to your automatic enrollment profile.
```

Comment:
I think this step is already covered above?

Reply:
Yeah it's a little repetitive. This is intentional because we're using this pattern elsewhere in the docs. For example the profiles docs here: https://fleetdm.com/docs/using-fleet/mdm-custom-macos-settings#step-2-upload-configuration-profile-to-fleet

I think we can leave it in but I'm happy to remove it if you think it's easier to understand w/o

Reply:
If it makes sense to you to keep it, then let's keep it.